docs/1-getting-started.md

```diff
@@ -4,7 +4,9 @@ Hush Line is a free and open-source, whistleblowing platform for organizations o
 
 ## Contents
 
-1. [Email Encryption](#email-encryption)
+1. Email Encryption
+   - [Option 1: Proton _(Recommended)_](#option-1-proton)
+   - [Option 2: Mailvelope and Gmail](#option-2-mailvelope--gmail)
```

Comment:
Should we mention gmail here? It doesn't seem like any of the instructions actually involve gmail specifically (barring the example smtp settings). Maybe "webmail" is a better option since that's more generally what mailvelope supports? I also wonder if we want to point folks to something like [GPGSuite](https://gpgtools.org/) which includes an extension for Apple's Mail client to support PGP decryption.

Reply:
Ah - good catch - Proton bundles forwarding in, and Mailvelope doesn't - I'll update the structure for more consistency

Reply:
The most popular PGP mail client is Thunderbird, so you could include instructions for that too potentially